pages/operator-address-select.js

```diff
@@ -22,25 +22,33 @@ const OperatorAddressLookup = (props) => (
           <Heading as="h2" size="MEDIUM">
             {props.t("Postcode")}
           </Heading>
-          <Paragraph className="operatorPostcodeDisplay" mb={0}>
-            {props.cumulativeFullAnswers.operator_postcode_find}
-          </Paragraph>
-          <Link
-            id="changeOperatorPostcode"
-            href={
-              props.editModeFirstPage
-                ? "/operator-address?edit=operator-address"
-                : "/operator-address"
-            }
-          >
-            {props.t("Change postcode")}
-          </Link>
+          <d1>
+            <dt>
+              <Paragraph className="operatorPostcodeDisplay" mb={0}>
+                {props.cumulativeFullAnswers.operator_postcode_find}
+              </Paragraph>
+            </dt>
+            <dd>
+              <Link
+                id="changeOperatorPostcode"
+                href={
+                  props.editModeFirstPage
+                    ? "/operator-address?edit=operator-address"
+                    : "/operator-address"
+                }
+              >
+                {props.t("Change postcode")}
+              </Link>
+            </dd>
+          </d1>
         </ContentItem.B_30_15>
         <ContentItem.B_20_20>
-          <SelectWithHeader
-            label={`${props.t("Select an address for")} ${
+          <Heading as="h2" size="MEDIUM" class="govuk-label govuk-label—l">
```

Comment:
I'd change these headings to large labels instead as suggested in the report. Same as the opening hours headings.

Reply:
hmm, my understanding from the report from the FSA report is that: it was a label so they suggested to change it to a header level 2 as it is descriptive and is introducing content, check below for the extract from the report:

‘Select an address for …..’ is visually displayed the same as ‘postcode’ which is a heading level 2, however, ‘Select an address for sa10 6sr’ is written as a span element, it is descriptive and is introducing content, therefore, this should be written as a heading level 2 to match the other heading on the page

Reply:
Yep, you're right, I think I just read the bottom section of the suggested solution - 'The ‘Select an address for…’ must have the GOV.UK Design System class suggested for labels: govuk-label govuk-label—l to avoid confusion ' - and mistook that as actually using a label, but can see you've added that as a class to the header.